modes/scheme-mode/lem-scheme-mode.asd

```diff
@@ -23,7 +23,7 @@
                (:file "repl")))
 
 (defmethod perform :after (operation
-                           (system (eql (find-system "lem-scheme-mode/repl"))))
+                           (system (eql (find-system "lem-scheme-mode"))))
```

Comment:
#<THREAD "main thread" RUNNING {10005185B3}>:
  Circular dependency:
     ((#<ASDF/LISP-ACTION:PREPARE-OP >
       . #<ASDF/SYSTEM:SYSTEM "lem-scheme-mode">)
      (#<ASDF/LISP-ACTION:PREPARE-OP >
       . #<ASDF/SYSTEM:SYSTEM "lem-scheme-mode/repl">)
      (#<ASDF/LISP-ACTION:LOAD-OP > . #<ASDF/SYSTEM:SYSTEM "lem-scheme-mode">)
      (#<ASDF/LISP-ACTION:LOAD-OP >
       . #<ASDF/LISP-ACTION:CL-SOURCE-FILE "lem-scheme-mode" "syntax-data">)
      (#<ASDF/LISP-ACTION:PREPARE-OP >
       . #<ASDF/LISP-ACTION:CL-SOURCE-FILE "lem-scheme-mode" "syntax-data">))

Reply:
Alright, I pushed a new commit which should handle this problem. However, I believe the system should (also) quickload `lem-scheme-mode/repl` when demanded. One way to do that is to create a stub for `start-scheme-repl` which calls `(maybe-quickload "lem-scheme-mode/repl")` and, if successful, calls the real `start-scheme-repl`.

Either that or really add the dependency on "async-process" (this would be the best solution, IMO).